### ISSUE TITLE
https on the blog is broke

### ISSUE THREAD

**Issue by ondrae** [codeforamerica/codeforamerica.org#911]
When trying to access our blog using https, the style sheets are messed up.
https://www.codeforamerica.org/blog/2014/10/30/the-civic-tech-issue-finder/
![screen shot 2015-02-04 at 3 17 07 pm](https://cloud.githubusercontent.com/assets/595778/6051882/eda5474e-ac80-11e4-947e-cea840202e4c.png)

Fortunately, we should be able to change all the style.codeforamerica.org links to use https and it should be fine. Same as in #529

**Comment by konklone** [comment]
I'm working on a PR for this.

**Comment by migurski** [comment]
@konklone :100:

**Comment by konklone** [comment]
I just filed #912 to fix everything I possibly could in here. But it looks like `/blog/` is a Wordpress app and managed separately, and `/library/` is something else and managed separately too. They both suffer from mixed-content blocking, but I don't know how to file a PR for them.

**Comment by migurski** [comment]
That’s right, they live elsewhere. The blog is in the [legacy-named cfawp2012 repo](https://github.com/codeforamerica/cfawp2012/) (it used to be our whole site) while the library is in [codeforamerica.org-library](https://github.com/codeforamerica/codeforamerica.org-library). A bit of guidance as we work on this and bump into oddball SSL stuff might be enough.
